### PR TITLE
change Harcoded strings and margins in fragment_media_details.xml 

### LINKED ISSUE #217: [Sub-issue] change Harcoded strings and margins in fragment_media_details.xml

### PR DESCRIPTION
**Pull request by kulloveth**
Fixes #217 
UI remains the same

### CODE REVIEW

File: app/src/main/res/layout/fragment_media_details.xml [hunk 1]
```diff
@@ -38,7 +38,7 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:adjustViewBounds="true"
-            android:minHeight="200dp"
+            android:minHeight="@dimen/dimens_200"
```
Comment (ujjwalagrawal17) [inline app/src/main/res/layout/fragment_media_details.xml]: dimens_200 --> dimens_200dp

**Comment by ujjwalagrawal17** [comment]
@kulloveth can you resolve the conflicts ?

**Comment by kulloveth** [comment]
> @kulloveth can you resolve the conflicts ?

fixed

**Comment by ujjwalagrawal17** [comment]
Thanks @kulloveth

**Pull request merged**